### PR TITLE
[FIX] storage_backend_ftp: use full path in ``move_files()``

### PR DESCRIPTION
**Pull request by SilvioC2C**

### CODE REVIEW

File: storage_backend_ftp/components/ftp_adapter.py [hunk 1]
```diff
@@ -150,8 +151,8 @@ def move_files(self, files, destination_path):
                     _logger.debug("destination %s is free", dest_file_path)
                 if result:
                     client.delete(dest_file_path)
-                # Move the file
-                client.rename(ftp_file, dest_file_path)
+                # Move the file using absolute filepaths
+                client.rename(fp(ftp_file), fp(dest_file_path))
```
Comment (simahawk) [inline storage_backend_ftp/components/ftp_adapter.py:155]: watch out: existing services could rely on the fact that the fullpath is passed already.
I would improve `_fullpath` to check if the base path is already there or not.

**Review by simahawk — CHANGES_REQUESTED** [review]

File: storage_backend_ftp/components/ftp_adapter.py (hunk 1 above)
Comment (SilvioC2C) [reply storage_backend_ftp/components/ftp_adapter.py:155]: @simahawk right, I'll update `_fullpath()` too to check that out

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
we should do the same for _sftp I think

**Comment by SilvioC2C** [comment]
@simahawk done here: https://github.com/OCA/storage/pull/148/commits/72ac0b61c13cc8ccf5c03a510ffddec246bec143

**Comment by simahawk** [comment]
where is this commit? :thinking: :smile:

**Comment by SilvioC2C** [comment]
@simahawk in the proper PR: https://github.com/OCA/storage/pull/148

**Comment by simahawk** [comment]
/ocabot merge patch

Could you backport both to v14 pls?

**Pull request merged**